guides/common/assembly_getting-started-with-ansible.adoc

```diff
@@ -8,7 +8,6 @@ Use this guide to configure {Project} to use Ansible, and then information about
 
 include::modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc[leveloffset=+1]
 include::modules/proc_importing-ansible-roles.adoc[leveloffset=+1]
-include::modules/proc_importing-ansible-variables.adoc[leveloffset=+1]
```

Comment:
This is the only time the `proc_importing-ansible-variables.adoc` file is included:
```
git grep -i "proc_importing-ansible-variables.adoc"
```
If it's no longer part of the "getting started with ansible" part, then we should delete it.